controls/V-72305.rb

```diff
@@ -37,11 +41,13 @@
 server_args = -s /var/lib/tftpboot"
   tag "fix_id": "F-78659r1_fix"
 
-  describe command('grep server_args /etc/xinetd.d/tftp') do
-    its('stdout.strip') { should match %r{^\s*server_args\s+=\s+-s\s\S+\s*$} }
-  end if package('tftp').installed?
-
-  describe "The TFTP package is not installed" do
-    skip "If a TFTP server is not installed, this is Not Applicable."
-  end if !package('tftp').installed?
+  if package('tftp-server').installed?
+    describe command('grep server_args /etc/xinetd.d/tftp') do
```

Comment:
technically we should use the `file` resource here and say `its('content') { should match .... }`